src/common/snippets/src/pass/mha_tokenization.cpp

```diff
@@ -344,22 +344,21 @@ ov::snippets::pass::TokenizeMHASnippets::TokenizeMHASnippets(const SnippetsToken
          */
 
         auto tokenize_transpose = [&](const std::shared_ptr<ov::opset1::Transpose>& transpose,
-                                      bool is_input_transposed, std::vector<int32_t> order,
+                                      bool is_input_transposed, size_t rank,
                                       const ov::NodeVector::const_iterator& pos) {
             // If Transpose has valid order for the Transpose fusing (ExplicitTransposeMatMulInputs pass call), tokenize him.
             // Otherwise, skip the Transpose.
+            if (rank <= 2)
```

Comment:
Please add a comment why we skip transpose tokenization for 2d cases

Reply:
Done